### PR TITLE
Add Manchester TMA Airpsace

### LINKED ISSUE #48: Define Manchester TMA Airpspace
MAN_CTR currently aliased to EGTT-N. Define Manchester TMA Airpsace correctly.

### PR DESCRIPTION
**Pull request by luke11brown**
Fixes #48 
Added Manchester TMA to FIRBoundaries.dat
Added Manchester TMA to VATSpy.dat
Removed Man TMA Ownership of EGTT-N

### CODE REVIEW

**Comment by nepetadosmil** [comment]
Pending review by VATUK Ops Dept.

File: VATSpy.dat [hunk 1]
```diff
@@ -17618,7 +17618,7 @@ EGTT-C|London ACC (Central) - London|LON_E|EGTT-C
 EGTT-C|London ACC (Central) - London|LTC_N|EGTT-C
 EGTT-C|London ACC (Central) - London|LTC_NW|EGTT-C
 EGTT-N|London ACC (North) - London|LON_N|EGTT-N
-EGTT-N|London ACC (North) - London|MAN|EGTT-N
+EGTM|Manchester TMA (Up to FL285)|MAN|EGTM
 EGTT-N|London ACC (North) - London|MAN_E|EGTT-N
```
Comment (hsugden) [inline VATSpy.dat:17622]: Should this line not be removed too?

**Comment by nepetadosmil** [comment]
Also please try keeping them in alphabetical order.

File: VATSpy.dat (hunk 1 above)
Comment (luke11brown) [reply VATSpy.dat]: Fixed in 708a0b1a41129bb55a6002fd1320df4258f35e96

**Comment by luke11brown** [comment]
Fixed in 708a0b1a41129bb55a6002fd1320df4258f35e96

**Comment by luke11brown** [comment]
Nestor, you merge only contained th VATSPy.dat removals.

None of the boundaries or Additions were actually added!

**Comment by nepetadosmil** [comment]
_Oops_

**Pull request merged**